Source/Fuse.Controls/Viewport.uno

```diff
@@ -123,14 +123,14 @@ namespace Fuse.Elements
 			get 
 			{ 
 				if (!HasVisualChildren) return null;
-				return  GetZOrderChild(0);
+				return FirstChild<Visual>();
```

Comment:
This may not be the same as the first child in Z order if there are multiple children. However, it does seem like it'd make more sense to have the actual first child instead.  And, since I think the code prevented multiple children this is probably fine.